packages/fluentui/react-northstar/src/components/Form/FormDropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import { compose, ComponentWithAs, ShorthandConfig } from '@fluentui/react-bindings';
+import { commonPropTypes } from '../../utils';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import Dropdown, { DropdownProps } from '../Dropdown/Dropdown';
+import FormFieldCustom, { FormFieldCustomProps, FormFieldCustomStylesProps } from './FormFieldCustom';
+
+interface FormDropdownOwnProps extends Omit<DropdownProps, 'styles'> {}
```

Comment:
Why in previous file it omits `variables` and here only `styles`? 🤔

Reply:
We don't omit `variables` some other files we are doing it with `acessibility` and `styles` but only added when it was complaining of conflict for different types